scrapers/koji.py

```diff
@@ -113,6 +113,71 @@ def get_build_tags(self, build_id):
 
         return self.teiid.query(sql=sql_query)
 
+    def get_tag_info(self, tag_name):
+        """
+        Query Teiid for tag_id of a tag and build_ids associated to it.
+
+        :param str tag_name: tag name
+        :return: a list of dictionaries
+        :rtype: list
+        """
+        sql_query = """
+            SELECT tags.id, tag_listing.build_id
+            FROM brew.tag_listing AS tag_listing
+            LEFT JOIN brew.tag AS tags ON tag_listing.tag_id = tags.id
+            WHERE tag_listing.active = True AND tags.name = '{}';
+            """.format(tag_name)
+
+        return self.teiid.query(sql=sql_query)
+
+    def get_build_info(self, build_ids):
+        """
+        Query Teiid for build info.
+
+        :param list build_ids: ID's of Koji builds
+        :return: a list of dictionaries
+        :rtype: list
+        """
+        sql_query = """
+                SELECT
+                events.time as creation_time,
+                build.completion_time,
+                build.epoch,
+                build.extra,
+                build.id,
+                brew.users.name as owner_name,
+                brew.users.krb_principal as owner_username,
+                package.name as package_name,
+                build.release,
+                build.start_time,
+                build.state,
+                build.task_id,
+                build.version
+            FROM build
+            LEFT JOIN events ON build.create_event = events.id
+            LEFT JOIN package ON build.pkg_id = package.id
+            LEFT JOIN brew.users ON build.owner = brew.users.id
+        """
+
+        sql_query += 'WHERE '
+        for index in range(len(build_ids)):
+            sql_query += 'build.id = {}'.format(build_ids[index])
+
+            if index != (len(build_ids) - 1):
+                sql_query += ' OR '
+
+        return self.teiid.query(sql=sql_query)
+
+    def process_tag_builds(self, tag_builds_info):
```

Comment:
Optional: My preference would be to not make this a separate method since it's so short and used only in one place